docs/package.json

```
@@ -13,10 +13,11 @@
 	"author": "Microsoft and contributors",
 	"scripts": {
 		"build": "concurrently npm:build:api npm:build:md-magic && npm run hugo",
-		"build:api": "npm run download:api && npm run build:api-documentation",
+		"build:api": "npm run download:api && npm run build:api-documentation && npm run build:redirects",
 		"build:api-documentation": "node ./api-markdown-documenter/index.js",
 		"build:md-magic": "markdown-magic",
 		"build:md-magic:code": "markdown-magic --files \"**/*.md\" !docs --workingDirectory ..",
+		"build:redirects": "node ./build-redirects.js",
```

Maybe

```suggestion
		"build:redirects": "shx cp data/versions.json api/fallback/versions.json",
```

Then add [shx](https://www.npmjs.com/package/shx) as a dep since there isn't already a copyfiles or cpy dependency you could use.